### PR TITLE
Soap cleans microwaves

### PR DESCRIPTION
**Pull request by rockwithagun**
A small patch that allows soap to clean dirty microwaves.
im coder
EDIT: Took your suggestions

### CODE REVIEW

**Comment by scrdest** [comment]
Could and should do what @Naxodile suggest, yeah.

File: config/admins.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@
 # Ranks can be anything defined in admin_ranks.txt           ~Carn   #
 ######################################################################
 
+rockwithagun - Host
```
Comment (Naxodile) [inline config/admins.txt]: Hm

Comment (rockwithagun) [inline config/admins.txt]: Shit, I cannot into Git.

**Comment by rockwithagun** [comment]
Closed until I can get my stuff in order.

**Pull request closed without merging**